client/my-sites/sidebar/manage-menu.jsx

```diff
@@ -119,6 +120,23 @@ class ManageMenu extends PureComponent {
 				wpAdminLink: 'edit-comments.php',
 				showOnAllMySites: false,
 			},
+			{
+				name: 'plugins',
+				label: translate( 'Plugins' ),
+				capability: 'manage_options',
+				queryable: ! isAtomicSite,
+				config: 'manage/plugins',
+				link: '/plugins',
+				paths: [ '/extensions', '/plugins' ],
+				wpAdminLink: 'plugin-install.php?calypsoify=1',
+				showOnAllMySites: true,
+				buttonLink: ! isAtomicSite ? '/plugins/manage' : '',
```

Comment:
We're missing the site slug, this will take us to the all sites plugin manage view which is disorienting

Reply:
> We're missing the site slug

Fixing.